frontend/appflowy_tauri/src/appflowy_app/constants/document/config.ts

```diff
@@ -1,15 +1,25 @@
 import { Align, BlockConfig, BlockType, SplitRelationship } from '$app/interfaces/document';
 import { randomEmoji } from '$app/utils/document/emoji';
 
+export const DELTA_BLOCK_TYPE = [
+  BlockType.TextBlock,
+  BlockType.HeadingBlock,
+  BlockType.TodoListBlock,
+  BlockType.BulletedListBlock,
+  BlockType.NumberedListBlock,
+  BlockType.QuoteBlock,
+  BlockType.CalloutBlock,
+  BlockType.ToggleListBlock,
+  BlockType.CodeBlock,
+  BlockType.EquationBlock,
```

Comment:
Does the math equation block contain delta?

Reply:
I will delete this variable